src/main/java/com/google/sps/data/Destination.java

```diff
@@ -0,0 +1,114 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.Set;
+
+// Represents a destination submitted by a user.
+public class Destination {
+
+  public enum Tag {
+    FOOD,
+    SPORT,
+    TOURIST,
+    HISTORICAL,
+    ART,
+    FAMILY,
+    UNDEFINED;
+  }
+
+  public enum Obscurity {
+    EASY,
+    MEDIUM,
+    HARD,
+    UNDEFINED;
+  }
+
+  private String name;
+  private LatLng location;
+  private String city;
+  private String description;
+  private ArrayList<com.google.sps.data.Riddle> riddles = new ArrayList<>();
```

Comment:
If two classes are in the same package (in this case, `com.google.sps.data`), you don't need to prefix the class name like this, nor do you need to add an import statement at the top.  DestinationDataServlet, by contrast, is in the `com.google.sps.servlets` package, so you _do_ need to add the import statement in that case.

```suggestion
  private ArrayList<Riddle> riddles = new ArrayList<>();
```